src/control/server/config/server.go

```diff
@@ -573,21 +547,58 @@ func (cfg *Server) validateMultiServerConfig(log logging.Logger) error {
 			}
 			seenBdevSet[dev] = idx
 		}
+	}
 
+	return nil
+}
+
+// validateEngineFabric ensures engine configuration parameters are valid.
+//
+// In addition to validating the fabric provider, check to see if the pinned
+// NUMA node was provided in the configuration. If it was provided, validate
+// that the NUMA node is correct for the given device. An error from
+// cfgEngine.Fabric.GetNumaNode() means that no configuration was provided in
+// the YML. Because this is an optional parameter, this is considered non-fatal.
```

Comment:
Just a nit but I'd be inclined to move this explanation out of the documentation header and put it next to the applicable code instead.